example/Dockerfile

```diff
@@ -42,7 +42,7 @@ ENV DOCKER_IMAGE="cloudposse/atmos"
 ENV DOCKER_TAG="latest"
 
 # Geodesic banner
-ENV BANNER="infrastructure"
+ENV BANNER="atmos"
```

Comment:
This change is fine. However, as long as you are updating the Dockerfile, please delete lines 62-70 (installing `saml2aws`, `assume-role`, and `vendir`), since those are not current best practice.

Reply:
removed `saml2aws` and `assume-role`.
`vendir` is needed in the example to get the components for testing